tests/test_shorthand.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2023 MosaicML Streaming authors
```

Comment:
Wondering if it is a good idea to move the `test_util.py` and `test_shorthand.py` under `tests/util` directory?

Reply:
done